clients/drcachesim/tracer/raw2trace.h

```diff
@@ -191,6 +191,26 @@ struct instr_summary_t final {
     byte packed_ = 0;
 };
 
+/**
+ * Functions for encoding memtrace data headers. Each function returns the number of bytes
+ * the write operation required: sizeof(trace_entry_t). The buffer is assumed to be
+ * sufficiently large.
+ */
+struct instruction_converter_t {
```

Comment:
"instru" in online_instru_t, etc. stands for "instrumentation", not "instruction": these entries are not specific to instructions.

Reply:
Oh. How about "conversion_utils_t" or "trace_markers_t"?

Reply:
I'm not sure what's being "converted"? And "marker" here refers to a specific entry type. How about trace_entry_writer_t or trace_metadata_writer_t or sthg.

Reply:
Sounds good - probably the "metadata" one is the closest.